src/js/utils/ui.js

```diff
@@ -107,16 +107,13 @@ function initInteractionListeners(ui) {
         const { pageX, pageY } = getCoords(e);
 
         ui.dragged = false;
-        ui.lastStart = now();
```

Comment:
Just out of curiosity, why is this redundant?

Reply:
It's already set by the preClickHandler, and is only used by the clickHandler, in order to prevent a click action in the event of longPress.